lib/pushmi_pullyu/swift_deposit.rb

```diff
@@ -0,0 +1,58 @@
+require 'openstack'
+
+class PushmiPullyu::SwiftDepositer
+
+  def initialize(connection: nil, container: nil)
+    raise 'conection can not be nil' if connection.nil?
+    raise 'container can not be nil' if container.nil?
+
+    user         = connection[:username]
+    pass         = connection[:password]
+    tenant       = connection[:tenant]
+    endpoint     = connection[:endpoint]
+    service_type  = 'object-store'
+    auth_method   = 'password'
+    auth_version = connection[:auth_version]
+
+    @swift_connection = OpenStack::Connection.create(username: user,
+                                                     api_key: pass,
+                                                     auth_method: auth_method,
+                                                     auth_url: "#{endpoint}/auth/#{auth_version}",
```

Comment:
Would it make sense to just have an `auth_url` option in the defaults? Instead of having both `endpoint`/`auth_version` ones?